Makefile

```diff
@@ -81,6 +81,10 @@ deploy: manifests kustomize
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=antrea-operator webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	cp config/crd/bases/operator.antrea.vmware.com_antreainstalls.yaml config/crd/operator.antrea.vmware.com_antreainstalls.yaml
+	cp config/crd/bases/operator.antrea.vmware.com_antreainstalls.yaml bundle/manifests/operator.antrea.vmware.com_antreainstalls.yaml
+	cp config/crd/bases/operator.antrea.vmware.com_antreainstalls.yaml deploy/kubernetes/operator.antrea.vmware.com_antreainstalls_crd.yaml
```

Comment:
Maybe the `crd/bases/...` usually is not used directly, I suggest `cp config/crd/operator.antrea.vmware.com_antreainstalls.yaml deploy/kubernetes/operator.antrea.vmware.com_antreainstalls_crd.yaml` instead. Just like the target `make install`. [https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/Makefile#L69](https://github.com/vmware/antrea-operator-for-kubernetes/blob/main/Makefile#L69)